docs/Course Wiki/ECE Course Offerings/ECE313.md

```diff
@@ -0,0 +1,83 @@
+
+## Course Tips
+
+There is a course textbook written by Bruce Hajek, which also contains many practice problems. The lectures often fall behind and are forced to skip content in order to catch up to other sections - thus it is highly recommended to read the course textbook. Many students have gotten As by only reading the textbook and never going to lecture. It is a great textbook.
+
+Probability is known for never following intuition. Be careful!
+
+The course moves very fast, as it has to cram a lot of content into very little time. The weekly homework is your opportunity to really sit down and learn the material, so it is recommended to do them fully on your own and only go to office hours if you have been stuck for more than an hour. The office hours right before the homework is due are often crowded - instead, attempt the homework early and go to the office hours right after homework is due. You'll probably be able to get a professor to yourself.
+
+Make sure you show your work on exams for some partial credit.
+
+## Life After
+
+Probability theory forms the basis of communications, control systems, theoretical computer science, and areas of other fields within electrical and computer engineering. Not surprisingly, ECE313 is a prerequisite for upper level technical electives in many of these areas, such as [ECE361 - Digital Communications](ECE361.md), [ECE418 - Introduction to Image and Video Processing](ECE418.md), [ECE438 - Communication Networks](ECE438.md), [ECE459 - Communications Systems](ECE459.md), [ECE449 - Maching Learning](ECE449.md), etc. For students seeking to broaden their intellectual (and possibly career) horizons, probability theory is applicable to modeling in many areas such as finance and economics. If you wish to pursue graduate studies in the aforementioned disciplines, you should consider [MATH347 - Fundamentals of Math](../MATH%20Course%20Offerings/MATH347.md) and [MATH444 - Elementary Real Analysis](../MATH%20Course%20Offerings/MATH444.md).
+
+## Infamous Topics
+
+- Functions of random variables: The math can get pretty nasty. However, all this really boils down to is managing inequalities. Make sure you know how to manage inequalities.
+- Joint random variables: It can get confusing as there are a lot of variables going on. It is recommended to develop your own personal system of variables (for example, reserve u and v for the original coordinate system only) and stick to this system for the entire semester.
+- Minimum Mean Square Error Estimation: There are a lot of symbols everywhere. Write down each case (constant, linear, free) on your cheatsheet, and forget about it until the problem on the exam asks for it.
+
+## Resources
+
+Professor Hajek's textbook is awesome. You can learn the entire course from it. Use it.
```

Comment:
link to book if possible? if not can we link to other resources?

Reply:
I will link it; but do you have any other resources?